tests/vmi_hook_sidecar_test.go

```diff
@@ -228,6 +234,123 @@ var _ = Describe("[sig-compute]HookSidecars", decorators.SigCompute, func() {
 	})
 })
 
+var _ = Describe("[sig-compute]HookQEMUArgsSidecar", decorators.SigCompute, func() {
+
+	var err error
+	var virtClient kubecli.KubevirtClient
+
+	var vmi *v1.VirtualMachineInstance
+
+	BeforeEach(func() {
+		virtClient = kubevirt.Client()
+
+		vmi = tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskAlpine))
+		vmi.ObjectMeta.Annotations = RenderValidQEMUArgsSidecar(hooksv1alpha1.Version)
+	})
+
+	Describe("[vendor:cnv-qe@redhat.com][level:component] VMI definition", func() {
+		getVMIPod := func(vmi *v1.VirtualMachineInstance) (*k8sv1.Pod, bool, error) {
+			podSelector := tests.UnfinishedVMIPodSelector(vmi)
+			vmiPods, err := virtClient.CoreV1().Pods(vmi.GetNamespace()).List(context.Background(), podSelector)
+
+			if err != nil {
+				return nil, false, fmt.Errorf("could not retrieve the VMI pod: %v", err)
+			} else if len(vmiPods.Items) == 0 {
+				return nil, false, nil
+			}
+			return &vmiPods.Items[0], true, nil
+		}
+
+		Context("with QEMU Args hook sidecar", func() {
+			It("should successfully start with hook sidecar annotation", func() {
+				By("Starting a VMI")
+				vmi, err = virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(nil)).Create(context.Background(), vmi)
+				Expect(err).ToNot(HaveOccurred())
+				libwait.WaitForSuccessfulVMIStart(vmi)
+			})
+
+			It("should successfully start with hook sidecar annotation for v1alpha2", func() {
+				By("Starting a VMI")
+				vmi, err = virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(nil)).Create(context.Background(), vmi)
+				vmi.ObjectMeta.Annotations = RenderValidQEMUArgsSidecar(hooksv1alpha2.Version)
```

Comment:
Probably you wanted to set the annotations first and then create the VMI.

Reply:
:man_facepalming:

So, the test was using hooks version `v1alpha1` which makes this test the exactly same as the previous one.
I'll be fixing this. Thanks!